vembrane/__init__.py

```diff
@@ -33,6 +37,7 @@ def get_annotation_keys(header: VariantHeader,) -> List[str]:
     return []
 
 
+@lru_cache(maxsize=32)
```

Comment:
as I said, we'll have to run this on actual (large) vcf files to find a good value